src/app/app-routing.module.ts

```diff
@@ -1,20 +1,24 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { HomeComponent } from '@app/file-manager';
+import { HomeComponent, LoginComponent } from '@app/file-manager';
 
 const routes: Routes = [
   {
     path: '',
-    redirectTo: 'home',
+    redirectTo: 'login',
     pathMatch: 'full'
   },
     {
     path: 'home',
     component: HomeComponent
+    },
+  {
+    path: 'login',
+    component: LoginComponent
```

Comment:
It's not clear to me how `LoginComponent` is used here. It looks like login works via `AppComponent.onLoginClick()` which is called from the `Sign In` button. I tried removing `LoginComponent` and its associated routes and it didn't seem to affect the login code behavior.

Reply:
The idea of `LoginComponent` is basically to deliver a previous step to access any work space download list without showing an empty list. Here we can notify if a user whether is allowed to access or it should register to Firecloud in order to access our app. 

At the moment this is empty since is to be implemented, but i can remove it from this branch if you consider its unnecessary right now.

Reply:
I think it's better to wait until it's needed to add it. You can make a story for the requirement you're thinking of and attach the code to the JIRA if you want to make sure the work isn't lost.